filters/openpolicyagent/internal/envoy/skipperadapter.go

```diff
@@ -25,7 +29,7 @@ func AdaptToExtAuthRequest(req *http.Request, metadata *ext_authz_v3_core.Metada
 				Http: &ext_authz_v3.AttributeContext_HttpRequest{
 					Host:    req.Host,
 					Method:  req.Method,
-					Path:    req.URL.Path,
+					Path:    buildPathWithQueryParams(req),
```

Comment:
Wouldn't this be even simpler: 

```suggestion
					Path:    req.URL.RequestURI(),
```

Reply:
You mean use `req.URL.RequestURI()`?
That would discard the path set by any setPath() filter in the chain and use the original request path.

Reply:
But it would work, wouldn't it? https://github.com/golang/go/blob/af0c40311e2ee33ecd24971257606f42a49cf593/src/net/url/url.go#L1165

It uses `EscapedPath()` internally which will use `Path` if it deviates from being the decoded form of `RawPath`

Reply:
Yeah, this simplifies things.
I made a wrong assumption previously that this would conflict with the setPath() filter.